<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientVideoWidget</class>
 <widget class="QWidget" name="ClientVideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientVideoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="videoToggleButton">
       <property name="toolTip">
        <string>Enable/disable video</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/ic_videocam_white_48dp.png</normaloff>
         <normalon>:/ic_videocam_off_white_48dp.png</normalon>:/ic_videocam_white_48dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fullscreenButton">
       <property name="toolTip">
        <string>Enter/leave fullscreen mode</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/ic_fullscreen_white_48dp.png</normaloff>
         <normalon>:/ic_fullscreen_exit_white_48dp.png</normalon>:/ic_fullscreen_white_48dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="toolTip">
        <string>Exit</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/ic_close_white_48dp.png</normaloff>:/ic_close_white_48dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>248</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
